open-craft/edx-platform
# Applies Patch Ironwood : Prevent reverse tab-nabbing

Patch Ironwood : Prevent reverse tab-nabbing

(cherry picked from commit 54a65f7744897329c0debf270ed3ca643d7147d1)

**JIRA tickets**: [SE-1733](https://tasks.opencraft.com/browse/SE-1733) and all its related issues.

**Discussions**: https://github.com/edx/edx-platform/pull/21635

**Sandbox URL**: TBD - sandbox is being provisioned.

* LMS: https://pr194.sandbox.stage.opencraft.hosting/
* Studio: https://studio.pr194.sandbox.stage.opencraft.hosting/

**Merge deadline**: ASAP

**Testing instructions**:

1. Using the sandbox, run through our [manual instance checklist](https://gitlab.com/opencraft/documentation/private/blob/master/checklists/manual_instance_test.md) to ensure that pages display ok, and that links and tabs work as expected.

**Author notes and concerns**:

1. Am not suggesting that we test to see that the security issue itself is fixed -- edX have provided this patch, so they have tested this.

**Reviewers**
